### ISSUE TITLE
Timecodes for "ML Zoomcamp 6.11 - Homework Solution"

### ISSUE THREAD

**Issue by alexeygrigorev** [DataTalksClub/machine-learning-zoomcamp#165]
Youtube video: https://www.youtube.com/watch?v=WoXSOyzcIbU

**Comment by dimzachar** [comment]
title is ML Zoomcamp 2021 - Homework #6 Solution

**Comment by dimzachar** [comment]
0:00:11 - Solving homework on tree-based regression model.
0:04:00 - Data preprocessing and decision tree regression in Python.
0:07:42 - Creating a Decision Tree Regressor and Visualizing it.
0:10:56 - Creating and evaluating a random forest regressor model.
0:14:27 - Creating and training a random forest regressor model.
0:18:34 - Showing improvement plateau in estimator, around 120.
0:21:29 - Explanation of iterative model training process.
0:24:45 - Using warm start to add trees and save time.
0:27:46 - Parameter tuning for random forest model. Plotting optimal values.
0:31:55 - Review: Explaining model stability and extracting feature importance.
0:35:34 - Feature importance demonstrated using decision trees and random forest.
0:38:47 - Review: Discussing feature importance and tuning parameters for XGBoost.
0:42:23 - Comparing model performance for different learning rates.
0:45:56 - Comparing convergence of models with different learning rates.
0:49:06 - Choosing model based on speed vs. accuracy trade-off.
0:52:09 - Homework review, ask questions, enjoy your project.

**Comment by amitfrancis** [comment]
Updated timecodes. Thanks!
